lib/stringify.js

```diff
@@ -56,6 +56,8 @@ var isNonNullishPrimitive = function isNonNullishPrimitive(v) {
         || typeof v === 'bigint';
 };
 
+var scSym = Symbol('SideChannel');
```

Comment:
We support many engines that do not have symbols.

Reply:
emmm....it was my poor consideration, I meant to define a unique symbol to identify the side-channel, and Symbol is good idea.